moveit_common/cmake/moveit_package.cmake

```
@@ -32,6 +32,9 @@ macro(moveit_package)
   find_package(ament_cmake REQUIRED)
   ament_package_xml()
 
+  # Enable backward_ros on every moveit package
+  find_package(backward_ros REQUIRED)
```

It doesn't make much difference for a well-setup structure, but QUIET would be enough here I guess?

```suggestion
  find_package(backward_ros QUIET)
```

---

I agree